app/mailers/student_mailer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+#
+# Mailer class for user e-mails
+class StudentMailer < ApplicationMailer
+  default from: 'no-reply@vesta.app' # SHOULD WE HAVE AN ADMIN ADDRESS?
```

Comment:
I also think it can be replaced with our eventual config model, so I'll leave a comment to that effect.